src/main/kotlin/blackjack/domain/Cards.kt

```diff
@@ -0,0 +1,47 @@
+package blackjack.domain
+
+class Cards(initialCards: List<Card> = emptyList()) {
+    val possibleResults
+        get() = getAllPossibleResults()
+
+    private val _cards = mutableListOf<Card>()
+    val cards
+        get() = _cards.toList()
+
+    init {
+        initialCards.forEach {
+            add(it)
+        }
+    }
+
+    fun add(card: Card) {
+        _cards.add(card)
+    }
+
+    fun addAll(servedCards: Cards) {
+        _cards.addAll(servedCards.cards)
+    }
+
+    private fun getAllPossibleResults(): List<Int> {
+        val results = mutableListOf<Int>()
+        calculateRecursive(cards, results)
+
+        return results
+    }
+
+    private fun calculateRecursive(cards: List<Card>, results: MutableList<Int>, idx: Int = 0, sum: Int = 0) {
+        if (idx > cards.lastIndex) {
+            results.add(sum)
+            return
+        }
+
+        val card = cards[idx]
+        if (card.cardNumber.isMultiple) {
+            card.cardNumber.multiNumbers?.forEach {
```

Comment:
Card에게 직접 메세지를 던지는 건 어떨까요?

```suggestion
        if (card.isMultiple) {
            card.multiNumbers?.forEach {
```